testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/SettingsMenuBaseActivity.kt

```diff
@@ -50,53 +57,60 @@ abstract class SettingsMenuBaseActivity : BaseActivity() {
     }
 
     private fun showAppSettings(): Boolean {
-        val settingsDialog = LayoutInflater.from(this)
-            .inflate(R.layout.app_settings_dialog, null, false)
+        val settingsView = LayoutInflater.from(this)
+            .inflate(R.layout.settings_menu_base_activity, null, false)
 
-        edtAppId = settingsDialog.findViewById(R.id.app_id) as EditText
-        edtSubscriptionKey = settingsDialog.findViewById(R.id.subscription_key) as EditText
-        val switchTest = settingsDialog.findViewById<SwitchMaterial>(R.id.switchTest)
+        edtAppId = settingsView.findViewById(R.id.app_id) as EditText
+        edtSubscriptionKey = settingsView.findViewById(R.id.subscription_key) as EditText
+        val switchTest = settingsView.findViewById<SwitchCompat>(R.id.switchTest)
         edtAppId.setText(settings.appId)
         edtSubscriptionKey.setText(settings.subscriptionKey)
         edtAppId.addTextChangedListener(settingTextWatcher)
         edtSubscriptionKey.addTextChangedListener(settingTextWatcher)
         switchTest.isChecked = settings.isTestMode
         validateSetting()
 
-        renderAppSettingsDialog(settingsDialog, edtAppId, edtSubscriptionKey, switchTest)
+        renderAppSettingsScreen(settingsView, edtAppId, edtSubscriptionKey, switchTest)
         return true
     }
 
-    private fun renderAppSettingsDialog(
-        settingsDialog: View,
+    @SuppressLint("SetTextI18n")
+    private fun renderAppSettingsScreen(
+        settingsView: View,
         appId: EditText,
         subscriptionKey: EditText,
-        switchTest: SwitchMaterial
+        switchTest: SwitchCompat
     ) {
-        val dialog = AlertDialog.Builder(this)
-            .setTitle("${resources.getString(R.string.lb_app_settings)} - Build " +
-                    resources.getString(R.string.miniapp_sdk_version) + " - " +
-                    resources.getString(R.string.build_version))
-            .setView(settingsDialog)
-            .setPositiveButton(R.string.action_save, null)
-            .setNegativeButton(android.R.string.cancel, null)
-            .create()
-
-        dialog.setOnShowListener { _dialog ->
-            btnSave = (_dialog as AlertDialog).getButton(AlertDialog.BUTTON_POSITIVE)
-            btnSave.setOnClickListener {
-                val pb = settingsDialog.findViewById<View>(R.id.pb)
-                pb.visibility = View.VISIBLE
-                updateSettings(appId.text.toString(), subscriptionKey.text.toString(), switchTest.isChecked,
-                    pb, _dialog)
-            }
+        title = resources.getString(R.string.lb_app_settings)
 
-            _dialog.getButton(AlertDialog.BUTTON_NEGATIVE).setOnClickListener {
-                _dialog.dismiss()
-            }
+        setContentView(settingsView)
+
+        val infoView = settingsView.findViewById<TextView>(R.id.textInfo)
+        infoView.text = "Build " + resources.getString(R.string.miniapp_sdk_version) + " - " +
```

Comment:
It's possible to dismiss the `resources` here, `getString` can be used directly.